test/features/binary_build.feature

```diff
@@ -0,0 +1,18 @@
+@binary
+Feature: Use binary build to deploy the service
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+
+  @springboot
+  Scenario: Deploy jbpm-springboot-example using binary build
```

Comment:
Do we support only springboot for binary build?

Reply:
We support quarkus too.
For both quarkus and springboot the process will be same (upload built project to OpenShift), so I test it just on one example. I chose springboot because it is faster.

Reply:
maybe add a small comment why this choice so we don't have that question again in 6 months XD